demos/README.md

```diff
@@ -144,7 +144,13 @@ cd build
 ```sh
 make
 ```
-
+or  
+Use this simple command which takes default build type `Release`:
+```sh
+mkdir build && cd build  
+cmake ..  
+make
+```
```

Comment:
I don't quite understand the purpose of this addition. These are exactly the same commands as above, except without the `-DCMAKE_BUILD_TYPE=Release` argument.

Reply:
for this CMAKE_BUILD_TYPE STREQUAL "" condition, CMAKE_BUILD_TYPE setted as Release In CMakeList. so we don't require to add `-DCMAKE_BUILD_TYPE=Release` explicitly for release Configuration

Reply:
True, but does that warrant creating a duplicate set of instructions? It's not that difficult to add `-DCMAKE_BUILD_TYPE=Release`, even if it is redundant.